src/AppInstallerCLICore/ExecutionArgs.h

```diff
@@ -218,10 +219,30 @@ namespace AppInstaller::CLI::Execution
             return types;
         }
 
+        // If the user passes the same value multiple times inside a MultiQuery, operations will be repeated
+        // Since there currently is not a way to include search options within a MultiQuery, processing duplicates
+        // does not make sense within a single invocation
+        void MakeMultiQueryContainUniqueValues()
+        {
+            auto itr = m_parsedArgs.find(Type::MultiQuery);
+            
+            // If there is not a value in MultiQuery, or there is only one value, it is already presumed to be unique
+            if (itr == m_parsedArgs.end() || itr->second.size() == 1)
+            {
+                return;
+            }
+
+            std::unordered_set<std::string> queryStrings;
+            for (auto query : itr->second)
+            {
+                queryStrings.insert(query);
+            }
+            m_parsedArgs[Type::MultiQuery].assign(queryStrings.begin(), queryStrings.end());

```

Comment:
This has possibility to change the execution order, but I guess it's ok?

Reply:
I specifically used an unordered set to hopefully avoid that; If there needs to be a strong preservation of order, then I suppose it might be possible to check whether or not the add to the set was successful, and if it was, emplace it into a new vector, then assign the new vector as the value of the arg.  However, I felt this was a cleaner way of doing it.

If it's okay with the team, I'd like to move forward with this approach first, and if ordering becomes an issue, it can be addressed as part of a separate issue

Reply:
I'm personally ok with this approach. Just wanted to ask about your thought. btw, set will almost always break the order. If we need to preserve order, we'll have to create a new vector and dedupe ourselves (we can dedupe case insensitively too).

Reply:
I think it will be best to let this one as-is, and I'll do a follow up to preserve order by doing the new vector

Reply:
Since the build is failing I took the time to do this now rather than separately